phys/module_mp_fast_sbm.F

```diff
@@ -6,6 +6,20 @@ SUBROUTINE SBM_fast
          dummy = 1
       END SUBROUTINE SBM_fast
       END MODULE module_mp_fast_sbm
+
+! Stub modules
+module module_mp_SBM_BreakUp
+end module module_mp_SBM_BreakUp
+
+module module_mp_SBM_Collision
+end module module_mp_SBM_Collision
+
+module module_mp_SBM_Auxiliary
+end module module_mp_SBM_Auxiliary
+
+module module_mp_SBM_Nucleation
+end module module_mp_SBM_Nucleation
+
 #else
```

Comment:
This is related to 
#if( BUILD_SBM_FAST != 1)
Can you remind me when this stub is used if it is in any of our builds?

Reply:
From what I could tell, it is used only when double-precision builds are requested as the subsequent code has a lot of mismatched complex types
The setting is [done in configure](https://github.com/wrf-model/WRF/blob/3dd1546ba425b5e77d682dd92253738416b3cb0a/configure#L602C8-L602C92)